src/apps/sources/__test__/utils.test.ts

```diff
@@ -1,9 +1,9 @@
 import { getSourceTypeFromPreviousPath } from "../utils";
 
 describe("getSourceTypeFromPreviousPath", () => {
-  it("should return public sources for /sources/", () => {
+  it("should returnAll Public Concepts for /sources/", () => {
```

Comment:
Ditto on not needing "All Public Concepts" here

Reply:
@ibacher thanks for the review. Let me fix the proposed changes.